tox.ini

```diff
@@ -7,6 +7,8 @@ envlist = py38
 deps =
 	pytest
 	black
+	mock-firestore
+	-rrequirements.txt
```

Comment:
is this correct? `-rrequirements.txt` without the spaces?

Reply:
Yeah I googled it lol

Reply:
It was in the official docs

Reply:
interesting. thanks for checking